### ISSUE TITLE
Improve repository and documentation

### ISSUE THREAD

**Issue by ceyhunsen** [chainwayxyz/clementine#204]
### Proposal Description

As we approach testnet launch, there will be a rise to number of people visiting this repo each day. Meaning we need a good looking readme and documentation.

Current readme still has the old information about old program flow. Also, we don't have a single binary that does a single thing but multiple binaries that does multiple things. There are more information that necessarily don't need to be there.

Readme should ideally have a brief introduction to what Clementine is and how to run basic binaries.

We can have a `doc/` directory that holds advanced usage information for the power users. This directory can include stuff like:

- How to run tests
- How to configure servers
- Configuration file options (related to #175)

TL;DR: Keep readme simple and have a `doc/` directory that includes advanced material.

**Comment by ozankaymak** [comment]
I think you are on point; however, we need a finalized design for people to get a grasp of what's going on. Therefore, I am afraid this issue will be open for a while.

**Comment by ceyhunsen** [comment]
We don't necessarily need to finalize documentation today, though. How we test and run our binaries hasn't changed for months now and seems like it is not going to change anytime soon.

But readme has incorrect steps and old information. At the very least, old and incorrect information should be deleted from readme. Adding simple documentation about current testing steps could be the icing on the cake, too.

It will be easier to write complete documentation later if we write simple steps now.
